sharpy/solvers/staticcoupled.py

```diff
@@ -194,7 +195,8 @@ def run(self):
                     struct_forces = \
                         self.correct_forces_generator.generate(aero_kstep=self.data.aero.timestep_info[self.data.ts],
                                                                structural_kstep=self.data.structure.timestep_info[self.data.ts],
-                                                               struct_forces=struct_forces)
+                                                               struct_forces=struct_forces,
+                                                               ts=0)
                 self.data.aero.timestep_info[self.data.ts].aero_steady_forces_beam_dof = struct_forces
```

Comment:
similar changes will be needed in `staticcoupledrbm.py` which also uses the polars

Reply:
Thanks for spotting! Fixed in https://github.com/ImperialCollegeLondon/sharpy/pull/217/commits/fcd30c718e9fa58ee8063be62507c72165fabbad